examples/hotrod-containerized/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+hotrod:
+  image: "jaegertracing/hotrod"
+  ports:
+    - "8080:8080"
+    - "8081:8081"
+    - "8082:8082"
+    - "8083:8083"
+  links:
```

Comment:
Do we want compose file for this at all? 

If so could you please remove links?

Reply:
From the author comments it does look like compose has some issues with ports. I would split it in a separate PR, assuming we even need it.

Reply:
@yurishkuro @pavolloffay  I used docker-compose as "quick and easy way" to create booth jaegertracing/all-in-one container and jaegertracing/hotrod.

Reply:
+1 let's go with dockerfile and then add compose if necessary